### DISCUSSION TITLE
How can I find which constraints are causing infeasibility?

### DISCUSSION THREAD

**Discussion by juliacaria** [anyoptimization/pymoo#586] [Q&A]
I'm trying to change how I tackle my problem, but now I'm facing an issue.
![image](https://github.com/anyoptimization/pymoo/assets/8332516/233976f8-d9f4-41ac-beb7-1ccf8181c750)

How can I find which of constraints are causing infeasiblity?

**Comment by blankjul** [comment]
The algorithm is not able to find a feasible solution. This could be because no exists (in the search space) or because the algorithm is not able to find it. A quick check is running an algorithm on the constraints only (you can use gradient or approximate gradient approaches for this as well).
